src/apscheduler/datastores/mongodb.py

```diff
@@ -281,10 +283,6 @@ async def acquire_schedules(self, scheduler_id: str, limit: int) -> list[Schedul
                     schedules.append(schedule)
 
                 if schedules:
-                    now = datetime.now(timezone.utc)
-                    acquired_until = datetime.fromtimestamp(
-                        now.timestamp() + self.lock_expiration_delay, now.tzinfo
-                    )
```

Comment:
Why remove this block? The query might take some time, and this code was there to take that into account.

Reply:
Makes sense. Reverted.